guides/v2.2/cloud/cdn/configure-fastly.md

```diff
@@ -355,40 +283,25 @@ Fastly options.
 Fastly also provides a series of [geolocation-related VCL features](https://docs.fastly.com/guides/vcl/geolocation-related-vcl-features)
 for customized geolocation coding.
 
-## Configure DNS for Fastly {#fastly-dns}
-
-You must complete these steps when you go live.
-
-After checking with your registrar about where to change your DNS settings,
-add a CNAME record for your website that points to the Fastly service:
-`prod.magentocloud.map.fastly.net`. If you use multiple hostnames for your site,
-you must add a CNAME record for each one.
+## DNS configuration {#dns}
 
-{:#info .bs-callout .bs-callout-info}
-This does not work for an [apex domain](https://blog.cloudflare.com/zone-apex-naked-domain-root-domain-cname-supp) (also referred to as a _naked_ domain). You must use a DNS
-provider that supports forwarding DNS queries to use an apex domain.
+You need to update the DNS configuration to  your:
```

Comment:
This ends with the word "your" and that makes me try to carry it on for each bullet. "...to your Set all necessary..."
Maybe there is a thought missing or interrupted here?